plaso/parsers/pe.py

```diff
@@ -44,6 +45,12 @@ class PEParser(interface.FileObjectParser):
   NAME = 'pe'
   DESCRIPTION = 'Parser for Portable Executable (PE) files.'
 
+  _PE_DIRECTORIES = [
```

Comment:
Why add the extra indirection here? This value is never reused.

Reply:
1. Since these definitions are static (do not change), so that they only need be created once
2. this makes the call to parse_data_directories, more compact